pkg/account/component.go

```diff
@@ -541,3 +543,51 @@ func (c *component) executeActions(ctx context.Context, actions []string) error
 
 	return err
 }
+
+func (c *component) CancelEmailChange(ctx context.Context) error {
```

Comment:
You also could have something like:

func (c *component) CancelEmailChange(ctx context.Context) error {
    return c.removeAttribute(ctx, constants.AttrbEmailToValidate)
}
func (c *component) CancelPhoneNumberChange(ctx context.Context) error {
    return c.removeAttribute(ctx, constants.AttrbPhoneNumberToValidate)
}
func (c *component) removeAttribute(...) error {
    ...
}